packages/suite/src/views/wallet/coinmarket/common/CoinmarketForm/CoinmarketFormInput/CoinmarketFormInputCountry.tsx

```diff
@@ -1,85 +1,57 @@
-    position: relative;
-    width: 20px;
-    height: 20px;
-    border-radius: ${borders.radii.full};
-    overflow: hidden;
-    margin-right: ${spacingsPx.xs};
-`;
-
-// reason: set cover size of flag for circle shape
-// eslint-disable-next-line local-rules/no-override-ds-component
-const FlagWrapper = styled(Flag)`
-    position: absolute;
-    top: 0;
-    left: 0;
-    width: 100%;
-    height: 100%;
-    object-fit: cover;
-    object-position: 50% 50%;
-`;
+import { Translation } from 'src/components/suite';
+import { TranslationKey } from 'src/components/suite/Translation';
 
 export const CoinmarketFormInputCountry = ({ label }: CoinmarketFormInputDefaultProps) => {
     const { control, setAmountLimits, defaultCountry } =
         useCoinmarketFormContext<CoinmarketTradeBuySellType>();
 
     return (
-        <>
-            <CoinmarketFormInputLabel label={label} />
-            <Controller
-                name={FORM_COUNTRY_SELECT}
-                defaultValue={defaultCountry}
-                control={control as Control<CoinmarketBuySellFormProps>}
-                render={({ field: { onChange, value } }) => (
-                    <Select
-                        value={value}
-                        options={regional.countriesOptions}
-                        onChange={selected => {
-                            onChange(selected);
-                            setAmountLimits(undefined);
-                        }}
-                        formatOptionLabel={(option: CountryOption) => {
-                            const labelParts = getCountryLabelParts(option.label);
-                            if (!labelParts) return null;
-
-                            return (
-                                <CoinmarketFormOption>
-                                    <FlagContainer>
-                                        <FlagWrapper country={option.value} />
-                                    </FlagContainer>
-                                    <CoinmarketFormOptionLabel>
-                                        {labelParts.text}
-                                    </CoinmarketFormOptionLabel>
-                                </CoinmarketFormOption>
-                            );
-                        }}
-                        data-testid="@coinmarket/form/country-select"
-                        isClearable={false}
-                        minValueWidth="160px"
-                        isSearchable
-                    />
-                )}
-            />
-        </>
+        <Controller
+            name={FORM_COUNTRY_SELECT}
+            defaultValue={defaultCountry}
+            control={control as Control<CoinmarketBuySellFormProps>}
+            render={({ field: { onChange, value } }) => (
+                <Select
+                    value={value}
+                    options={regional.countriesOptions}
+                    labelLeft={<Translation id={label as TranslationKey} />}
```

Comment:
Please, delete forcing type and fixing an issue in the image below
![obrazek](https://github.com/user-attachments/assets/26bcd6f1-db45-40c0-a463-1efd53a65676)